cmd/trade.go

```diff
@@ -166,10 +167,10 @@ func init() {
 		dataKey := model.MakeSortedBotKey(assetBase, assetQuote)
 		strat, e := plugins.MakeStrategy(sdex, tradingPair, &assetBase, &assetQuote, *strategy, *stratConfigPath, *simMode)
 		if e != nil {
-			log.Println()
-			log.Println(e)
+			l.Info("")
+			logger.Fatal(l, e)
```

Comment:
this shouldnot be a `logger.Fatal` because fatal means we will exit, which we don't want to do here -- see the next line where we want to delete all offers and then exit.

Reply:
Yikes, glad you caught that.